installation.md

```diff
@@ -30,12 +28,22 @@ There are various valid ways to install them. Be sure to install a recent versio
 
 You must install and start redis. There's no specific configuration to do for Miaou (it's preferable to configure redis to *not* save on disk, as we only use it as a session cache making it possible to restart miaou without the users noticing).
 
+## Debian & Co
+
 	sudo apt-get install redis-server
 
 There's several ways to start it. Here's the simplest one:
 
 	redis-server &
 
+## Arch linux
+
+	sudo pacman -S redis
+	sudo systemctl start redis # only start server for this session
+	sudo systemctl enable redis # will start server for every **future** session
```

Comment:
You actually can shrink these commands into one: `[sudo] systemctl enable --now redis[.service]`